index.js

```diff
@@ -97,7 +106,6 @@ function convertCrontabs() {
           ...newCrontabs.map((crontab, i) => ({
             schedule: Object.assign({}, schedule, {
               rate: `cron(${crontab})`,
-              name: schedule.name && `${schedule.name}-${i}`
```

Comment:
Should we always remove that naming here? If I understand, it's not supported when there are multiple rates specified

Reply:
Yup, that's correct, so if `newCrontabs` has more than one entry or if `schedule.rate` has more than one entry, the name property is invalid in current Serverless versions.

I can make it so that the naming still is applied when `schedule.rate` is a string to maintain the old behavior for older serverless versions.

Reply:
I think it would make sense to keep the behavior consistent for older versions

Reply:
@pgrzesik I've updated this now.